### PR TITLE
pipecontrol: init at 0.2.2

### PR DESCRIPTION
**Pull request by TilCreator**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/pipecontrol/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pipecontrol";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "portaloffreedom";
+    repo = pname;
+    rev = "v" + version;
```
Comment (fabaff) [inline pkgs/applications/audio/pipecontrol/default.nix:24]:
```suggestion
    rev = "v${version}";
```

File: pkgs/applications/audio/pipecontrol/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    kcoreaddons
+    ki18n
+    qttools
+    qtquickcontrols2
+  ];
+
+  meta = with lib; {
+    description = "Pipewire control GUI program in Qt (Kirigami2)";
+    homepage = "https://github.com/portaloffreedom/pipecontrol";
+    license = licenses.gpl3;
```
Comment (fabaff) [inline pkgs/applications/audio/pipecontrol/default.nix:46]: According to the [file header](https://github.com/portaloffreedom/pipecontrol/blob/master/src/systemdservice.cpp#L5-L7) is the license `GPLv3`.

```suggestion
    license = licenses.gpl3Only;
```

File: pkgs/applications/audio/pipecontrol/default.nix [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    rev = "v" + version;
+    sha256 = "sha256-BeubRDx82MQX1gB7GnGJlQ2FyYX1S83C3gqPZgIjgoM=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+    wrapQtAppsHook
+  ];
+  buildInputs = [
```
Comment (fabaff) [inline pkgs/applications/audio/pipecontrol/default.nix:33]:
```suggestion
  ];

  buildInputs = [
```

File: pkgs/applications/audio/pipecontrol/default.nix [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    pipewire
+    qtbase
+    kirigami2
+    kcoreaddons
+    ki18n
+    qttools
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/pipecontrol/default.nix:39]: Goes into nativeBuildInputs.

Comment (TilCreator) [reply pkgs/applications/audio/pipecontrol/default.nix:39]: Which of the possibly marked ones?
I'm unsure if only `qttools` or more are marked

Comment (SuperSandro2000) [reply pkgs/applications/audio/pipecontrol/default.nix:40]: yeah, only qttools.

**Pull request merged**